postgres_composite_types/__init__.py

```diff
@@ -192,11 +188,15 @@ def __new__(mcs, name, bases, attrs):
                 fields.append((field_name, field))
 
         # retrieve the Meta from our declaration
-        meta_obj = attrs.pop('Meta', object())
+        try:
+            meta_obj = attrs.pop('Meta')
+        except KeyError:
+            raise TypeError('%s has no Meta class' % (name,))
```

Comment:
Let's wrap `Meta` in quotes, so you know it's a name, and not a Metaclass. I misread it first time.